app/controllers/application_controller.rb

```diff
@@ -2,11 +2,13 @@ class ApplicationController < ArtsyAuth::ApplicationController
   protect_from_forgery with: :exception
   force_ssl if: :ssl_configured?
 
-  ALLOWED_ROLES = ['admin'].freeze
+  ADMIN_ROLE = 'marketing_db_admin'.freeze
+  ALLOWED_GRAVITY_ROLES = ['admin', ADMIN_ROLE].freeze
 
   def authorized_artsy_token?(token)
-    roles = user_roles(token)
-    success_or_die(roles)
+    @roles = user_roles(token)
+    @is_admin = admin?
```

Comment:
Looks like we want to make the distinction between gravity admin and bearden admin here. I'd suggest a more explicit naming to avoid confusion.